com.unity.render-pipelines.high-definition/Runtime/Sky/VisualEnvironment.cs

```diff
@@ -12,12 +12,19 @@ namespace UnityEngine.Rendering.HighDefinition
     public sealed class VisualEnvironment : VolumeComponent
     {
         /// <summary>Type of sky that should be used for rendering.</summary>
+        [Header("Sky")]
         public NoInterpIntParameter skyType = new NoInterpIntParameter(0);
         /// <summary>Type of clouds that should be used for rendering.</summary>
         public NoInterpIntParameter cloudType = new NoInterpIntParameter(0);
         /// <summary>Defines the way the ambient probe should be computed.</summary>
         public SkyAmbientModeParameter skyAmbientMode = new SkyAmbientModeParameter(SkyAmbientMode.Static);
 
+        /// <summary>Controls the global orientation of the wind relative to the X world vector.</summary>
+        [Header("Wind")]
+        public ClampedFloatParameter windOrientation = new ClampedFloatParameter(0.0f, 0.0f, 360.0f);
+        /// <summary>Controls the global wind speed in kilometers per hour.</summary>
+        public MinFloatParameter windSpeed = new MinFloatParameter(100.0f, 0.0f);
```

Comment:
I don't know if this is the default wind value we want for everything

Reply:
then the question is do we want a global wind parameter if different systems need different value. I think we will see when we have smoething other than the sky using this parameter